### ISSUE TITLE
TypeError: RegionalTimeseriesActivity.__init__() got an unexpected keyword argument 'files'

### ISSUE THREAD

**Issue by xgui3783** [FZJ-INM1-BDA/siibra-python#586]
as of 3d3b74798157f0e5762b0a2e75fe53425fb2a197 (used by siibra-api rc build)

sample code

```python
import siibra
feature = siibra.features.Feature._get_instance_by_id("cf0::RegionalBOLD::p:minds/core/parcellationatlas/v1.0.0/94c1125b-b87e-45e4-901c-00daee7f2579-290::r:Area hOc1 (V1, 17, CalcS) left::164b7564-1730-4b51-9c98-308005b620dd::2a993ff2650c1a95e9c331e6a33c4325")
feature.plot()
```

expected result: return plot object

actual result:

```
Loading preconfigured RegionalBOLD instances: 100%|████████████████████████████| 8/8 [00:00<00:00, 827.34RegionalBOLD/s]
Matching RegionalBOLD to Area hOc1 (V1, 17, CalcS) left: 100%|████████████████████| 1600/1600 [00:00<00:00, 4482.02it/s]
>>> f.plot()
[siibra:INFO] CompoundFeature.data averages the data of each element.
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/xiao/dev/projects/siibra-api/venv/lib/python3.10/site-packages/siibra/features/feature.py", line 779, in plot
    return self._get_merged_feature().plot(*args, **kwargs)
  File "/home/xiao/dev/projects/siibra-api/venv/lib/python3.10/site-packages/siibra/features/feature.py", line 785, in _get_merged_feature
    self._merged_feature_cached = self.feature_type._merge_elements(
  File "/home/xiao/dev/projects/siibra-api/venv/lib/python3.10/site-packages/siibra/features/tabular/regional_timeseries_activity.py", line 105, in _merge_elements
    merged = cls(
  File "/home/xiao/dev/projects/siibra-api/venv/lib/python3.10/site-packages/siibra/features/tabular/regional_timeseries_activity.py", line 273, in __init__
    RegionalTimeseriesActivity.__init__(self, **kwargs)
TypeError: RegionalTimeseriesActivity.__init__() got an unexpected keyword argument 'files'
```

**Comment by AhmetNSimsek** [comment]
I found the issue but, on another note, how you did obtained this feature ID?

**Comment by AhmetNSimsek** [comment]
The bold features are anchored at the parcellations so the query should be with a parcellation. But there is no `get_profile` as in connectivity features. I can add this functionality but could you remind me how siibra-api queries connectivity profiles? (or the flowchart)

**Comment by xgui3783** [comment]
do you mean that I shouldn't be able to get any bold features?

**Comment by AhmetNSimsek** [comment]
Fixed as of 1777ded9e68d28542ca4caf95f08f49ed3645c94
